src/main/java/no/sikt/sws/Prefixer.java

```diff
@@ -0,0 +1,164 @@
+            case INVALID:
+                throw new BadRequestException("resourceIdentifier [" + resourceIdentifier + "] is invalid");
+            default:
+                throw new IllegalStateException("Unexpected value: " + resourceIdentifier);
+        }
+    }
+
+
+    private static String mappingkBody(String workspacePrefix, String gatewayBody) {
+        logger.debug(workspacePrefix);
+        return gatewayBody;
+    }
+
+    private static String searchBody(String workspacePrefix, String gatewayBody) {
+        logger.debug(workspacePrefix);
+        return gatewayBody;
+    }
+
+    // replace {index} with {workspace}-{index} from responseBody
+    public static String indexesBody(String workspacePrefix,String gatewayBody) {
+
+        var node =  string2JsonNode(gatewayBody);
+
+        nodeUpdated(workspacePrefix,"aliases",node);
+        nodeUpdated(workspacePrefix,"settings/index",node);
+
+        return node.toPrettyString();
+    }
+
+    private static void nodeUpdated(String workspacePrefix, String nodeName, JsonNode node) {
+        if (node.has(nodeName)) {
+            var objectNode = (ObjectNode) node.get(nodeName);
+            var names = new ArrayList<String>();
+            objectNode.fieldNames().forEachRemaining(names::add);
+            names.forEach(name -> {
+                objectNode.set(workspacePrefix + "-" + name,objectNode.get(name));
+                objectNode.remove(name);
+            });
+            ((ObjectNode) node).set(nodeName,objectNode);
+        }
+    }
+
+    protected static String bulkBody(String workspacePrefix, List<JsonNode> gatewayBulkBody) {
+
+        return gatewayBulkBody.stream().map(item -> {
+            String indexName;
+            if (item.has("index")) {
+                indexName = item.get("index").get(INDEX).textValue();
+            } else if (item.has("delete")) {
+                indexName = item.get("delete").get(INDEX).textValue();
+            } else if (item.has("update")) {
+                indexName = item.get("update").get(INDEX).textValue();
+            } else if (item.has("create")) {
+                indexName = item.get("create").get(INDEX).textValue();
+            } else {
+                return item.toString();
+            }
+            var workspaceIndexName = String.format("\"%s-%s\"", workspacePrefix, indexName);
+            indexName = String.format("\"%s\"",  indexName);
+
+            return item.toString().replaceAll(indexName,workspaceIndexName);
+
+        }).collect(Collectors.joining("\n")) + "\n";
+    }
+
+    protected static String aliasBody(String workspacePrefix, String gatewayAliasBody) {
+        return gatewayAliasBody
+                .replaceAll("(\"index\".*?\")(.+?\")","$1" + workspacePrefix + "-$2")
+                .replaceAll("(\"alias\".*?\")(.+?\")","$1" + workspacePrefix + "-$2");
+    }
+
+    private static List<JsonNode> getBulkBody(String bulkBody) {
```

Comment:
You dont really get a bulk body, but you cnvert it to jsons. Would a better name be "convertBulkBodyToJsonList" or smething similar?